README.md

```diff
@@ -1,75 +1,81 @@
+* Compare Notebooks side-by-side, with diffs for source, markdown, output, and execution count cells
+* Share Notebook checkpoints or diffs with persistent links
+* Download Notebook checkpoints directly from Neptune or Jupyter 
 
-Technically Neptune + Jupyter integration is implemented as a neptune-notebooks Python package which includes:
-* Standard Jupyter extension - The extension is written in Javascript and is visible in the web-based Jupyter UI. 
-* JupyterLab extension - The extension is written in Javascript and is visible in the web-based Jupyter UI.
-* A very basic CLI (command line interface) for uploading notebooks.
+![Jupyter Notebook rendered in Neptune](https://docs.neptune.ai/img/app/notebook_view.png)
 
 ## Resources
 
 * [Documentation](https://docs.neptune.ai/tools/jupyter/overview/)
-* [Example notebook in the Neptune app](https://app.neptune.ai/neptune-ai/credit-default-prediction/n/exploring-application-table-ac75c237-1630-4109-b532-dd125badec0e/ca1df3be-b2e4-4b26-99d6-b7e98a3d4273)
+* [Example notebook in Neptune](https://app.neptune.ai/neptune-ai/credit-default-prediction/n/exploring-application-table-ac75c237-1630-4109-b532-dd125badec0e/ca1df3be-b2e4-4b26-99d6-b7e98a3d4273)
 
-## Example
+## Setup
 
-```python
-# On the command line, install the neptune-notebooks extension
-pip install neptune-notebooks
+Install the extension:
 
-# Enable the extension for Jupyter
-jupyter nbextension enable --py neptune-notebooks
+```sh
+pip install neptune-notebooks
+```
 
+Enable the extension for Jupyter:
 
-# Install the Neptune client
-pip install neptune-client
+```sh
+jupyter nbextension enable --py neptune-notebooks
 ```
 
 In your Jupyter Notebook environment, some Neptune items appear in your toolbar.
-1. Click the Neptune icon to connect with Neptune.
-2. Click **Upload** to create a first notebook checkpoint in Neptune.
-3. In the Jupyter notebook, import Neptune and start a run:
+
+1. To connect with Neptune, click the Neptune **Configure** button.
+2. Enter your [Neptune credentials](https://docs.neptune.ai/setup/setting_credentials/#finding-your-credentials).
+3. To register the notebook in Neptune, click **Upload**.
+
+This uploads a first checkpoint of the notebook. Every time you start a Neptune run in the notebook, a checkpoint is uploaded automatically.
+
+For detailed instructions, see the [Neptune documentation](https://docs.neptune.ai/tools/jupyter/installing_extension/).
+
+## Usage
+
+In a notebook cell, import neptune and start a run:
 
 ```python
-import neptune.new as neptune
-run = neptune.init(
-    api_token="NEPTUNE_API_TOKEN",
-    project="WORKSPACE_NAME/PROJECT_NAME"
-)
+import neptune
 
-# Log metadata
+run = neptune.init_run()
+```
+
+Log model-building metadata that you care about:
+
+```python
 run["f1_score"] = 0.66
+```
 
-# Stop the run
-run.stop()
+When you're done with the logging, stop the run:
 
-# See notebook snapshots in Neptune under "Source code"
+```python
+run.stop()
 ```
 
```

Comment:
Let's mention where they can find the uploaded notebook - in the "Source Code" namespace, and in the "Notebooks" tab in project.